docs/dev/README.md

```diff
@@ -0,0 +1,102 @@
+| [infrastructure operator](#infrastructure-operator) | light / heavy | :x: | this should be valid to any test related with assisted-installer |
+| [dev-scripts](##dev-scripts) | heavy | :x: | all-road environment. Anything should work here |
+
+## podman
+[Detailed documentation](../../deploy/podman/README.md)  
+[REST-API V2 - Getting Started Guide](../docs/user-guide/rest-api-getting-started.md)
+
+In this scenario the goal is to run small tests with the assisted-installer REST API. This can be done in a local environment by starting the assisted-service with podman.
+
+During assisted-image-service startup, the images configured in the `OS_IMAGES` and `RELEASE_IMAGES` variables present in the [deploy/podman/configmap.yml](../../deploy/podman/configmap.yml) file will be downloaded. By default, these values ​​contain Openshift versions from 4.9 onwards. You may want to define a smaller set to improve resource usage. For example:
+```yaml
+OS_IMAGES: '[{"openshift_version":"4.16","cpu_architecture":"x86_64","url":"https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.16/4.16.3/rhcos-4.16.3-x86_64-live.x86_64.iso","version":"416.94.202406251923-0"}]'
+RELEASE_IMAGES: '[{"openshift_version":"4.16","cpu_architecture":"x86_64","cpu_architectures":["x86_64"],"url":"quay.io/openshift-release-dev/ocp-release:4.16.10-x86_64","version":"4.16.10","default":true}]' 
+```
+
+We can start this local environment using make:
+```shell
+make deploy-onprem
+```
+Once finished, we can see the containers running:
+```shell
+$ podman ps
+
+CONTAINER ID  IMAGE                                                      COMMAND               CREATED        STATUS        PORTS                                                                   NAMES
+487e6c1bdb9a  localhost/podman-pause:4.9.3-1708357294                                          4 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  7c18ebd0915a-infra
+8479b5eb8a8d  quay.io/sclorg/postgresql-12-c8s:latest                    run-postgresql        4 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  assisted-installer-db
+ffb9013c4fab  quay.io/edge-infrastructure/assisted-installer-ui:latest   /deploy/start.sh      3 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  assisted-installer-ui
+100c865abfd6  quay.io/edge-infrastructure/assisted-image-service:latest  /assisted-image-s...  3 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  assisted-installer-image-service
+78924b68f7af  quay.io/edge-infrastructure/assisted-service:latest        /assisted-service     3 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  assisted-installer-service
+```
+For testing we can use `curl`:
```

Comment:
WDYT about mentioning and linking aicli? Maybe something like:

"For testing we can use `curl` (or aicli for more complex flows)"

Reply:
Yes, why not? I usually use `aicli` for SaaS but it is also possible to use it with a local cluster.
Do I need to provide an example of `aicli` or is linking it enough?